R1 reduce jargon & repitition (#297)
SORTEE-Github-Hackathon/manuscript: content/02.introduction.md

Since its launch in 2005, Git has become the leading version control system in software development and in other disciplines that similarly require collaboration and community contributions, such as scientific research [@doi:10.1109/MS.2012.61].
Although the understanding of basic concepts of Git (such as commit, push, pull, checkout; see [Box 1](#definitions)) is necessary, the GitHub web-based platform and its integrated development environments (such as GitHub Desktop) allow users to perform most repository and data management operations without opening Git command-line sessions, making these functionalities available to users who are less familiar with software development.

Version control involves tracking the state of the files in a folder, called a "repository" (see [Box 1](#definitions)), through the evolution of a project.
A basic workflow using git and GitHub is to 1) make changes to files in a repository on your computer, 2) record related changes in files as a "commit" (see [Box 1](#definitions)) along with a message describing the changes, 3) synchronize commits with GitHub (see "push" in [Box 1](#definitions)) so that the repository on the web is up to date. 

> Collaborators (or you on a different computer) can then view your changes online and synchronize them to their own computers (see "pull" in [Box 1](#definitions)).

Reviewer: ```suggestion
Collaborators can then view your changes online and synchronize them to their own computers (see "pull" in [Box 1](#definitions)).
```

I don't think we need the "or you" part

> Commits act like snapshots and one can revisit the state of the project to any commit to view or undo changes.

Reviewer: ```suggestion
Commits act like snapshots and one can view or even revert the state of the project to any previous commit.
```

> If the modified files are plain text (e.g. computer code or .txt files) only the differences from the previous commit are recorded, allowing you frequently commit changes without causing the size of the project to grow excessively.

Reviewer: ```suggestion
If the modified files are plain text (_e.g._, computer code or .txt files) only the differences from the previous commit are recorded, allowing you frequently commit changes without causing the size of the project to grow excessively.
```

> This provides a safe and less cluttered alternative to frequently making full copies of documents at different points in their evolution (e.g. analysis.R, analasys_v2.R, analysis_FINAL.R).

Reviewer: ```suggestion
This provides a safe and less cluttered alternative to frequently making full copies of documents at different points in their evolution (_e.g.,_ analysis.R, analasys_v2.R, analysis_FINAL.R).
```

> More technical details about the use of Git and GitHub are beyond the scope of this paper (but see ref, ref, ref).

Reviewer: ```suggestion
More technical details about the use of Git and GitHub are beyond the scope of this paper (but see [@doi:10.1371/journal.pcbi.1004947; @doi:10.1371/journal.pcbi.1004668]).
```

Adding in citations

Version control involves tracking the state of the files in a folder, called a "repository" (see [Box 1](#definitions)), through the evolution of a project.
A basic workflow using git and GitHub is to 1) make changes to files in a repository on your computer, 2) record related changes in files as a "commit" (see [Box 1](#definitions)) along with a message describing the changes, 3) synchronize commits with GitHub (see "push" in [Box 1](#definitions)) so that the repository on the web is up to date. 
Collaborators (or you on a different computer) can then view your changes online and synchronize them to their own computers (see "pull" in [Box 1](#definitions)).
Commits act like snapshots and one can revisit the state of the project to any commit to view or undo changes.

> If the modified files are plain text (e.g. computer code or .txt files) only the differences from the previous commit are recorded, allowing you frequently commit changes without causing the size of the project to grow excessively.

Reviewer 2: add a comma and "to": If the modified files are plain text (e.g. computer code or .txt files), only the differences from the previous commit are recorded, allowing you to frequently commit ...

This provides a safe and less cluttered alternative to frequently making full copies of documents at different points in their evolution (e.g. analysis.R, analasys_v2.R, analysis_FINAL.R).

> More technical details about the use of Git and GitHub are beyond the scope of this paper (but see ref, ref, ref).

Reviewer 2: could cite [doi.org/10.1371/journal.pcbi.1004947](https://doi.org/10.1371/journal.pcbi.1004947) and [doi.org/10.1371/journal.pcbi.1004668](https://doi.org/10.1371/journal.pcbi.1004668)

Reviewer 3: perhaps this one as well: Kalliamvakou, Eirini, et al. "Open source-style collaborative development practices in commercial projects using github." 2015 IEEE/ACM 37th IEEE international conference on software engineering. Vol. 1. IEEE, 2015.
